apps/o2-blocks/package.json

```diff
@@ -22,12 +22,12 @@
 		"build": "check-npm-client && calypso-build editor='./src/editor.js' --env.WP"
 	},
 	"dependencies": {
-		"@wordpress/block-editor": "3.8.0",
-		"@wordpress/blocks": "6.13.0",
-		"@wordpress/components": "9.3.0",
-		"@wordpress/editor": "9.13.0",
-		"@wordpress/element": "2.12.0",
-		"@wordpress/i18n": "3.10.0",
-		"classnames": "2.2.6"
+		"@wordpress/block-editor": "^3.8.0",
+		"@wordpress/blocks": "^6.13.0",
+		"@wordpress/components": "^9.3.0",
+		"@wordpress/editor": "^9.13.0",
+		"@wordpress/element": "^2.12.0",
+		"@wordpress/i18n": "^3.10.0",
+		"classnames": "^2.2.6"
```

Comment:
`classnames` hasn't been moved to the root `package.json` as a pinned dependency. Given that o2 blocks is an "app" should this dependency still be pinned?

Reply:
I think we're ok not pinning these versions anywhere for now, since we already have the ranges here guaranteeing we won't get incompatible versions. Team Calypso is working on a migration to `yarn` where we won't be pinning at all, just ranges, and they're hoping to get it ready in the next few days.